recomp-verify.py

```diff
@@ -73,78 +86,201 @@ def verify_single_process(spec, cfg, cust, naive, verbose):
     os.chdir(orig_dir)
 
 def verify_capture_output(spec, cfg, pdir, *args):
-    shutil.copy(spec, pdir+"/")
-    shutil.copy(cfg, pdir+"/")
-    #os.chdir(pdir)
-    #write("out.log","")
+    # Captures output of the verification process
+    shutil.copy(spec, pdir + "/")
+    shutil.copy(cfg, pdir + "/")
 
     cmd_args = ["java", "-Xmx7g", "-jar", tool, spec, cfg]
     for arg in args:
         cmd_args.append(arg)
 
-    # uses TLC to run the one-component case
+    # Uses TLC to run the one-component case
     if pdir == "mono":
         cmd_args = ["java", "-Xmx7g", "-jar", tlc, "-deadlock", "-workers", "1", "-config", cfg, spec]
 
-    #naive_log = open("out.log","w")
-    #process = subprocess.Popen(cmd_args, stdout=naive_log.fileno())
-    #process.wait()
-    #return open("naive/out.log").read().rstrip()
-    #ret = subprocess.run(cmd_args, capture_output=True, text=True)
-
-    cd_args = ["cd",pdir]
+    cd_args = ["cd", pdir]
     cmd = " ".join(cd_args) + "; " + " ".join(cmd_args)
     ret = subprocess.run(cmd, capture_output=True, text=True, shell=True)
 
+    # Print the return code and stdout/stderr for debugging
+    print(f"Command exited with return code: {ret.returncode}")
+    print(f"Command stdout: {ret.stdout}")
+    print(f"Command stderr: {ret.stderr}")
+
     return ret.stdout.rstrip()
 
-def run_multi_verif(dest_dir, spec, cfg):
-    executor = concurrent.futures.ThreadPoolExecutor(max_workers=8)
-    futures = []
+def get_all_output(dest_dir, subdirs):
+    """
+    Print contents from log files
+    """
+    if True:
+        print("\n--- Log Outputs ---")
+        for subdir in subdirs:
+            log_file = os.path.join(dest_dir, subdir, f"{subdir}.log")
+            if os.path.exists(log_file):
+                with open(log_file, 'r') as f:
+                    content = f.read().strip()
+                    if content:  # Only print non-empty logs
+                        print(f"\n- Contents of {subdir}/{subdir}.log -:\n")
```

Comment:
is these debugging prints? if so, please delete or make a part of verbose printing. same goes for the rest of the prints here